package.json

```diff
@@ -42,14 +48,22 @@
   },
   "scripts": {
     "start": "react-app-rewired start",
-    "build": "react-app-rewired build",
+    "build": "yarn localize && node scripts/build.js",
```

Comment:
what is the difference between "node scripts/build.js" and "react-app-rewired"?

Reply:
Hmm... I didn't mean to change to `node scripts/build.js`.

Reply:
Fixed.